tests/tt_eager/python_api_testing/unit_testing/misc/test_moreh_sum.py

```diff
@@ -277,7 +277,7 @@ def test_moreh_sum_enable_cache(input_shape, dim, device, use_program_cache):
     for i in range(2):
         passing = moreh_sum(input_shape, dim, keep_batch_dim[i], use_provide_output[i], False, device)
         assert passing
-    assert device.num_program_cache_entries() == 1
+    assert device.num_program_cache_entries() == 2
```

Comment:
why?

Reply:
Because there was a missing attribute in the manually added reflection methods